### ISSUE TITLE
Rename "Contribute" into "Participate"

### ISSUE THREAD

**Issue by robertoostenveld** [Radboud-University/osc-nijmegen#46]
This is for discussion...

**Comment by KristijanArmeni** [comment]
Good idea, 'contribute' is perhaps narrower ('contribute sth tangible to the project') whereas 'participate' might relate to a broader type of activities. I will bring this up for discussion at the next occasion.
